source/auth/auth.rst

```diff
@@ -970,6 +970,8 @@ authMechanismProperties=PROPERTY_NAME:PROPERTY_VALUE,PROPERTY_NAME2:PROPERTY_VAL
 gssapiServiceName (deprecated)
 	An alias for ``authMechanismProperties=SERVICE_NAME:mongodb``.
 
+If any of the auth related options are specified in the connection string without a value, e.g. ``mongodb://localhost/admin?authSource=``, the default value for the option should be used.
```

Comment:
Update the changes section and last modified.

Reply:
Per discussion in #driver-devs, I think this should just specify that drivers should raise an error if `authSource` is an empty string. We also talked about opening separate tickets to (a) clarify how empty values in a connection string are parsed (in connection string spec) and (b) specify validation rules for other string options that might receive an empty value (in the URI options spec).

@jstewart-mongo: Neither the PR nor the commit reference [SPEC-1514](https://jira.mongodb.org/browse/SPEC-1514). Please make sure it's at least in the commit message so this change gets linked back to the JIRA issue (in both directions).

Reply:
@rozza I have updated the changes section and last modified.
@jmikola I have updated the commit message to reference the ticket.
I created a new subsection to describe what drivers should do when `authSource` is specified with an empty string.